browser/src/control/jsdialog/Widget.TreeView.ts

```diff
@@ -1161,6 +1166,10 @@ class TreeViewControl {
 		}
 
 		if (entries && entries.length === 0) this.makeEmptyList(data, builder);
+
+		// we need to provide a way for making the treeview control focusable
+		// when no entry is selected
+		if (level === 1 && !hasSelectedEntry) this._container.tabIndex = 0;
```

Comment:
can we create method for that:
```
makeTreeViewFocusable(focus: boolean) {
if (focus)
    this._container.tabIndex = 0;
else
    this._container.removeAttribute('tabindex');
}
```

and use here and in the `selectEntry()` - that will show the conection between the two instructions

Reply:
Done